cw_4/src/Pages/ProductDetails/ProductDetails.jsx

```diff
@@ -1,608 +0,0 @@
-                    >
-                      {Product && Product[0].rating}
-                    </CircularProgressLabel>{" "}
-                  </CircularProgress>
-                  <Box paddingLeft={"10px"}>
-                    <Star rating={Product && Product[0].rating} />
-                  </Box>
-                </Box>
-                <Box w="40%">
-                  <Flex color="green">
-                    5{" "}
-                    <Slider
-                      isReadOnly="true"
-                      defaultValue={20}
-                      colorScheme="green"
-                      marginLeft={"12px"}
-                    >
-                      <SliderTrack>
-                        <SliderFilledTrack />
-                      </SliderTrack>
-                    </Slider>
-                  </Flex>
-                  <Flex color="green">
-                    4{" "}
-                    <Slider
-                      isReadOnly="true"
-                      defaultValue={30}
-                      colorScheme="green"
-                      marginLeft={"12px"}
-                    >
-                      <SliderTrack>
-                        <SliderFilledTrack />
-                      </SliderTrack>
-                    </Slider>
-                  </Flex>
-                  <Flex color="green">
-                    3{" "}
-                    <Slider
-                      isReadOnly="true"
-                      defaultValue={70}
-                      colorScheme="green"
-                      marginLeft={"12px"}
-                    >
-                      <SliderTrack>
-                        <SliderFilledTrack />
-                      </SliderTrack>
-                    </Slider>
-                  </Flex>
-
-                  <Flex color="red">
-                    2{" "}
-                    <Slider
-                      isReadOnly="true"
-                      defaultValue={30}
-                      colorScheme="red"
-                      marginLeft={"12px"}
-                    >
-                      <SliderTrack>
-                        <SliderFilledTrack />
-                      </SliderTrack>
-                    </Slider>
-                  </Flex>
-                  <Flex color="red">
-                    1{" "}
-                    <Slider
-                      isReadOnly="true"
-                      defaultValue={10}
-                      colorScheme="red"
-                      marginLeft={"12px"}
-                    >
-                      <SliderTrack>
-                        <SliderFilledTrack />
-                      </SliderTrack>
-                    </Slider>
-                  </Flex>
-                </Box>
-              </Flex>
-              <Text fontSize={"20px"} margin={"12px"} color="gray">
-                1334 Ratings & 114 Reviews
-              </Text>
-            </Box>
-          </Box>{" "}
-        </>
-      )}
-    </div>
-  );
-};
```

Comment:
The whole component is written really well, but can we split this component into smaller components to make it more readable and try adding more comments before each section to give brief about the section